backend/main.py

```
@@ -36,24 +36,25 @@
-# Mapping for manim models
-MANIM_MODEL_MAPPING = {
-    'Main Finetuned': 'finetuned',  # Maps to LoRA finetuned model
-    'Deepseek': 'base'             # Maps to base model
-}
+MANIM_MODEL_SERVER_URL = os.getenv("MODEL_SERVER_URL")
 
 # Configure CORS
 app.add_middleware(
     CORSMiddleware,
-    allow_origins=[f"http://localhost:{os.getenv('FRONTEND_PORT')}"],
+    allow_origins=[f"http://localhost:{os.getenv("FRONTEND_PORT")}"],
```

Pylint (3.3.7)

[error] Parsing failed: 'f-string: unmatched '(' (main, line 57)'

(E0001)